### ISSUE TITLE
videojs-webrtc-plugin - videojs-webrtc-plugin.min.js

### ISSUE THREAD

**Issue by fabiox23** [ant-media/Ant-Media-Server#6642]
Short question. Where can I find the file "videojs-webrtc-plugin.min.js"? It doesn't seem to be in the videojs-webrtc-plugin repository. Thanks.

**Comment by fabiox23** [comment]
Thanks, I found the js file after doing "npm install" and copied it to the server. However, the html sample code in the README.md file doesn't seem to work for me.  Apart from the video id which should be "my-video" and not "video-player", when I publish a stream with id "stream1", I get the following errors in the log console:

Failed to load resource: net::ERR_UNKNOWN_URL_SCHEME
WebSocket connection to 'ws://localhost:5080/LiveApp/websocket?target=edge' failed: 
initWebSocketConnection	@	￼videojs-webrtc-plugin.min.js:2

Any help please?

**Comment by rahul78275** [comment]
Thanks @fabiox23 for pointing out the typo 👍 (anyone from AMS team will update it).

Did you installed AMS on your local system ? Let me reproduce this at my end and will revert you soon.

**Comment by fabiox23** [comment]
Yes, I have AMS installed on the same server where the html sample code is running. The stream "stream1" definitely exists as I can play it with play.html

This is the code:

<html>
<head>
</head>
<body>
<script src="https://cdnjs.cloudflare.com/ajax/libs/video.js/7.21.6/video.min.js"></script>
<script src="/coachye/js/videojs-webrtc-plugin.min.js"></script>
<div id="video_container">
    <video id="my-video" width="960" height="540" class="video-js vjs-default-skin" controls>
        <source
           src="ws://localhost:5080/LiveApp/stream1.webrtc" iceServers = '[ { "urls": "stun:stun1.l.google.com:19302" } ]'
           >
      </video>
</div>
<script>
  var player = videojs('my-video');
</script>
</body>
</html>

**Comment by rahul78275** [comment]
![image](https://github.com/user-attachments/assets/3e854359-12d8-4f59-bebe-263c615eed8b)

I can play at my end, however i don't have ams installed on my local system.

If you are available and comfortable with a google meet call now. Please share the meeting url. We can resolve this together.

**Comment by fabiox23** [comment]
How do I send you a private message?

**Comment by rahul78275** [comment]
Here: rahul78275@gmail.com
